chat-server/src/services/llm.ts

```diff
@@ -31,91 +31,85 @@ export type OpenAiStreamingResponse = AsyncIterable<
 >;
 export type OpenAiAwaitedResponse = OpenAiChatMessage;
 
-export class OpenAiLlmProvider
-  implements LlmProvider<OpenAiStreamingResponse, OpenAiAwaitedResponse>
-{
-  private openAiChatClient: OpenAiChatClient;
+interface MakeOpenAiLlmParams {
```

Comment:
Minor, but we are using both "Params" and "Args" in the codebase and should probably unify on one.

Reply:
i believe Param is more accurate (see [this page](https://byjus.com/gate/difference-between-argument-and-parameter-in-c-and-c-plus-plus/#:~:text=or%20Actual%20Parameters.-,What%20is%20Parameter%3F,-The%20variables%20that)). where do we use args?